jenkins_pipelines/scripts/tests/test_maintenance_json_generator.py

```diff
@@ -138,6 +142,33 @@ def test_validate_and_store_results(self, mock_print):
         # cleanup
         remove(test_output_file)
 
+    def test_read_mi_ids_from_file(self):
+        file_ids: list[str] = read_mi_ids_from_file(_TEST_MI_IDS_FILE_PATH)
```

Comment:
Why exacly `_TEST_MI_IDS_FILE_PATH` is a global variable? Can we do it statically as it is with `test_output_file`?

Reply:
Especially if you already created this file, any other parameter would fail tests.